bedrock/firefox/templates/firefox/campaign/unfck/index.html

```diff
@@ -181,7 +181,7 @@ <h2 class="c-section-title u-visually-hidden">What we’re doing</h2>
         title='Here’s how Firefox unfcks the internet',
         ga_title='card-unfck',
         image_url=None,
-        link_url='',
+        link_url='https://blog.mozilla.org/firefox/how-to-unfck/',
```

Comment:
I'm not seeing UTM params added to this one, but they're on the other one... not sure what's up with that, maybe because it's a card?

Reply:
Oh is it missing `data-cta-text`? That's the only difference I see.